### ISSUE TITLE
Cicero CLI v0.11.1 is broken

### ISSUE THREAD

**Issue by dselman** [accordproject/cicero#324]
```
npm install -g @accordproject/cicero-cli
npm WARN deprecated nomnom@1.6.2: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN deprecated postinstall-build@5.0.3: postinstall-build's behavior is now built into npm! You should migrate off of postinstall-build and use the new `prepare` lifecycle script with npm 5.0.0 or greater.
/Users/dselman/.nvm/versions/node/v8.15.0/bin/cicero -> /Users/dselman/.nvm/versions/node/v8.15.0/lib/node_modules/@accordproject/cicero-cli/index.js
+ @accordproject/cicero-cli@0.11.1
updated 1 package in 3.552s
Dan-MacBook-Pro:latedeliveryandpenalty dselman$ cicero --version
module.js:550
    throw err;
    ^

Error: Cannot find module 'chai'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/dselman/.nvm/versions/node/v8.15.0/lib/node_modules/@accordproject/cicero-cli/node_modules/@accordproject/ergo-engine/lib/util.js:20:14)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

**Comment by dselman** [comment]
It looks like `ergo-engine` requires `Chai` but it is declared as a devDependency:
https://github.com/accordproject/ergo/blob/master/packages/ergo-engine/package.json#L39

We probably need to refactor this so that we don't require `Chai` at runtime.

**Comment by jeromesimeon** [comment]
Fix is in, see PR #327

**Comment by jeromesimeon** [comment]
With 0.11.1:
```bash-3.2$ npm install -g @accordproject/cicero-cli
npm WARN deprecated postinstall-build@5.0.3: postinstall-build's behavior is now built into npm! You should migrate off of postinstall-build and use the new `prepare` lifecycle script with npm 5.0.0 or greater.
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/cicero -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/index.js

> fsevents@1.2.7 install /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/fsevents
> node install

node-pre-gyp WARN Using request for node-pre-gyp https download 
[fsevents] Success: "/Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node" is installed via remote

> nunjucks@3.1.2 postinstall /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/nunjucks
> node postinstall-build.js src

+ @accordproject/cicero-cli@0.11.1
added 508 packages from 392 contributors in 37.746s
bash-3.2$ cicero --help
module.js:550
    throw err;
    ^

Error: Cannot find module 'chai'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/@accordproject/ergo-engine/lib/util.js:20:14)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

**Comment by jeromesimeon** [comment]
With the development version:
```
bash-3.2$ npm install -g @accordproject/cicero-cli@0.11.2-20190326183124 
npm WARN deprecated postinstall-build@5.0.3: postinstall-build's behavior is now built into npm! You should migrate off of postinstall-build and use the new `prepare` lifecycle script with npm 5.0.0 or greater.
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/cicero -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/index.js

> cucumber-expressions@6.0.1 postinstall /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/cucumber-expressions
> node scripts/postinstall.js

+ @accordproject/cicero-cli@0.11.2-20190326183124
added 57 packages from 204 contributors and updated 6 packages in 16.705s
bash-3.2$ cicero --help
Commands:
  parse     parse sample text using a template
  archive   archive a template directory
  execute   execute a clause with JSON request
  init      initialize a clause
  generate  generate code from the template model

Options:
  --help         Show help                                             [boolean]
  --version      Show version number                                   [boolean]
  --verbose, -v                                                 [default: false]

```

**Comment by jeromesimeon** [comment]
Fixed in Cicero 0.12:
```
bash-3.2$ npm install -g @accordproject/cicero-cli@0.12.0 
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/cicero -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/index.js

> fsevents@1.2.7 install /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/fsevents
> node install

node-pre-gyp WARN Using request for node-pre-gyp https download 
[fsevents] Success: "/Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node" is installed via remote

> cucumber-expressions@6.0.1 postinstall /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/cicero-cli/node_modules/cucumber-expressions
> node scripts/postinstall.js

+ @accordproject/cicero-cli@0.12.0
added 542 packages from 529 contributors and updated 5 packages in 39.394s
bash-3.2$ cicero --version
0.12.0
```
